packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -1400,14 +1401,29 @@ export class ColorPicker
     context.arc(x, y, radius, startAngle, endAngle);
     context.fillStyle = "#fff";
     context.fill();
+
     context.strokeStyle = "rgba(0,0,0,0.3)";
     context.lineWidth = outlineWidth;
     context.stroke();
 
+    const pattern = context.createPattern(this.getCheckeredBackgroundPattern(), "repeat");
```

Comment:
Can you move this into the `if` statement block below? `pattern` is created but only used if alpha will be applied.